src/OpenDiffix.Core.Tests/Analyzer.Tests.fs

```diff
@@ -415,6 +415,16 @@ type Tests(db: DBFixture) =
     analyzeQueryUntrusted "SELECT floor(age) from customers" |> ignore
     analyzeQueryUntrusted "SELECT age from customers" |> ignore
 
+  [<Fact>]
+  let ``Detect queries with disallowed bucket functions calls`` () =
+    ensureQueryFailsUntrusted
+      "SELECT round(2, age) from customers"
```

Comment:
let's make the second arg here valid, i.e. `2` or `2.0`, or make that the third tested case.

Reply:
If both arguments are constants, then the function call is replaced by a constant during normalization.

Reply:
aaah, right! that's good then. I'm not sure whether and how to test it then, up to you. In a way, it is good that it doesn't fall into some "strange" path and produce an irrelevant error message.